coding-concepts.js

```diff
@@ -53,7 +54,7 @@ class Learn {
   }
 }
 var learnStudent = new Learn("George")
-// console.log(learnStudent)
+console.log(learnStudent)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: i think this will log undefined because there is no kind of return or log statement.
+// b) Verify and explain: it returned { student: 'George', cohort: 'Delta', year: 2021 }. Because of constructor statement it returns the object and its contents
```

Comment:
correct because classes are not functional code. They are inert blueprints / plans that can be acted upon.